main.py

```diff
@@ -2,6 +2,44 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create(
```

Comment:
Checklist p.2 and 3. In this task is actually expected put initial data in tuples and create instances with iteration. 
`bulk_create` is nice but it is not exactly what we are looking for in this task